asyncio port scanner.py

```diff
@@ -0,0 +1,31 @@
+import time
+import asyncio
+
+
+async def connect_socket(ip: str, port: int):
+    """
+    The function try to connect a given socket to a given ip and port. If not
+    succeeded it raise an exception and print a massage.
+    :param ip: The ip for the socket to connect.
+    :param port: The port for the socket to connect.
+    """
+    try:
+        await asyncio.open_connection(ip, port)
+        print(f"port {port} found!")
+    except Exception as e:
+        print(e)
+        print(f"port {port} not found")
+
+
+async def main():
+    start_t = time.perf_counter()
+
+    await asyncio.gather(*[connect_socket('127.0.0.1', i)
```

Comment:
Use double quotes when using strings.
```suggestion
    await asyncio.gather(*[connect_socket("127.0.0.1", i)
```